bears/vcs/git/GitCommitBear.py

```diff
@@ -260,6 +265,17 @@ def check_body(self, body,
                                'Commit body lines should not exceed {} '
                                'characters.'.format(body_line_length))
 
+        if verify_email:
+            result_message = 'Body contains these invalid emails:\n'
+            invalid_emails = False
+            for line in body:
+                for email in re.findall(r'\w+@+\w+\.+\w*', line):
+                    if is_email(email, diagnose=True, check_dns=True).ERROR_CODES:
+                        invalid_emails = True
+                        result_message += ' ' + email + '\n'
```

Comment:
Line is longer than allowed. (82 > 79)

*LineLengthBear, severity NORMAL, section `linelength`.*